frontend/src/components/nav.js

```diff
@@ -133,6 +134,15 @@ class NavigationBar extends Component {
                   <a className="nav-bar-link pl-3">Dashboard</a>
                 </Link>
               </NavItem>
+              {this.state.isDirector && (
+                <>
+                  <NavItem>
+                    <Link prefetch href="/workspaces">
+                      <a className="nav-bar-link pl-3">Workspaces</a>
+                    </Link>
+                  </NavItem>
+                </>
```

Comment:
There's only one top level component here so I don't think we need the Fragment? Unless theres something I missing.